bundle/edu.gemini.ags/src/test/scala/edu/gemini/ags/impl/GemsStrategySpec.scala

```diff
@@ -633,6 +630,73 @@ class GemsStrategySpec extends Specification {
       val estimate = gemsStrategy.estimate(ctx, ProbeLimitsTable.loadOrThrow())(implicitly)
       Await.result(estimate, 1.minute) should beEqualTo(Estimate.GuaranteedSuccess)
     }
+
+    // Previously, before the mag 3 difference limit was imposed in guide stars in an asterism due to the NGS2 changes,
+    // MASCOT would have chosen the asterism:
+    // CWFS1: 289-128992, R=11.238
+    // CWFS2: 289-129007, R=14.006
+    // CWFS3: 289-129036, R=14.849
+    // However, this asterism is no longer valid as the maximum mag difference is 14.849 - 11.238 = 3.611 > 3.
+    "select CWFS guide stars being at maximum 3 mag difference for NGS2" in {
```

Comment:
Great that we test this

Reply:
I really wanted to add a test for the non-filtered asterisms as well, but that involved replicating a huge amount of the code that prepared the necessary parameters to MASCOT.